instrumented/integration/src/androidTest/kotlin/com/datadog/android/sdk/integrationtests/profiling/LightMemoryProfileForLogs.kt

```diff
@@ -31,6 +31,6 @@ internal class LightMemoryProfileForLogs {
                     mockServerRule.activity.logger.d("Test Crash", crash)
                 }
             }
-        }, threshold = 40)
+        }, threshold = 300)
```

Comment:
Instead of having those magic thresholds there, It'd be nice to have a comment above or below with the different values and why we changed them. eg: 
```kotlin 
    // 300 kB measured on an emulator with API XX on Bitrise
```